.github/CODEOWNERS

```diff
@@ -291,7 +291,7 @@
 /sdk/eventhub/                                  @serkantkaraca @jsquire
 
 # PRLabel: %Event Hubs %Functions
-/sdk/eventhub/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/ @pakrym @JoshLove-msft @jsquire
```

Comment:
![](https://media4.giphy.com/media/113RhN1oBm1yCc/giphy.gif)

Reply:
Thank you, @weshaggard.   There were no less than three of us poking at this for a bit to troubleshot and every single one of us overlooked the obvious typo.